src/test/unit_tests/test_agent_blocks_verifier.js

```diff
@@ -175,38 +190,63 @@ mocha.describe('mocked agent_blocks_verifier', function() {
             });
     });
 
-    mocha.it('should not verify blocks on deleted nodes', function() {
+    // mocha.it('should not verify blocks on deleted nodes', function() {
```

Comment:
This is an important test that checks the verifier not sending verification requests to nodes that are deleted.
This means that the verifier can get stuck in timeout loops and will attempt to verify blocks on non-existing nodes which will hurt it's performance badly.